### PR TITLE
Continental Mirrors

### PR DESCRIPTION
**Pull request by FuSoftware**
Added fixes to handle continental servers along with worldwide servers

### CODE REVIEW

File: src/downloader.cpp [hunk 1]
```diff
@@ -29,7 +29,7 @@ Downloader::loadMirrors()
 		if (reply->error() != QNetworkReply::NoError)
 			return;
 
-		QStringList tmp = page.split("mirrors in other parts of the world");
+        QStringList tmp = page.split("mirrors in other countries, but same continent");
```
Comment (JKSH) [inline src/downloader.cpp]: Please indent using tabs ('\t') instead of spaces.

File: src/downloader.cpp [hunk 2]
```diff
@@ -30,7 +30,7 @@ Downloader::loadMirrors()
 			return;
 
         QStringList tmp;
-        if page.contains("mirrors in other countries, but same continent")
+        if(page.contains("mirrors in other countries, but same continent"))
```
Comment (JKSH) [inline src/downloader.cpp]: Please add a space after the `if`.

File: src/downloader.cpp [hunk 3]
```diff
@@ -29,7 +29,18 @@ Downloader::loadMirrors()
 		if (reply->error() != QNetworkReply::NoError)
 			return;
 
-        QStringList tmp = page.split("mirrors in other countries, but same continent");
+        QStringList tmp;
```
Comment (JKSH) [inline src/downloader.cpp]: Please indent using tabs ('\t') instead of spaces.

File: src/downloader.cpp [hunk 4]
```diff
@@ -29,7 +29,18 @@ Downloader::loadMirrors()
...
+        if page.contains("mirrors in other countries, but same continent")
+        {
+            //Handles continental servers if existing
```
Comment (JKSH) [inline src/downloader.cpp]: Please add a space after the `//`.

File: src/downloader.cpp [hunk 5]
```diff
@@ -29,7 +29,18 @@ Downloader::loadMirrors()
...
+            tmp = page.split("mirrors in other countries, but same continent");
+        }
+        else
+        {
+            //Handles worldwide servers if no continental server was found
```
Comment (JKSH) [inline src/downloader.cpp]: Please add a space after the `//`.

**Review by JKSH — CHANGES_REQUESTED** [review]
Hi @FlorentUguet, thank you for improving the mirror extraction mechanism!

Please follow the style of the existing code:
- Indent using tabs ('\t') instead of spaces.
- Have a space after `if` and `\\`

Please also squash your 3 commits into 1, since they are part of the same change (it's ok to `--force` a push to this repo)

**Comment by FuSoftware** [comment]
Thanks for the info, I'm working on it and it should be done quickly

About \t : I use QtCreator on Window, and use the Tab key. I guess QtCreator uses spaces itself for that.

As for the squashing, looking at GitHub's docs seems to show that you'll have to squash the commits since you're the owner of the main branch. You'll have to squash my commits while merging with your master

Edit : QtCreator transforms the `\t` into 4 spaces itself. I am looking for some kind of setting, but it looks like I cannot fix that and only create more 4 spaces tabulations

Edit : Found how to solve that under Projects/Code Style/

**Comment by JKSH** [comment]
Thank you for your contribution!

**Pull request merged**